src/core/cubic.c

```diff
@@ -385,6 +388,10 @@ CubicCongestionControlOnDataAcknowledged(
         // Slow Start
         //
 
+        //
+        // TODO: CongestionWindow should only be allowed to grow up to SlowStartThreshold
```

Comment:
This is unrelated to the CUBIC bis FYI. The issue is that we intend to grow exponentially only up to SlowStartThreshold, but instead we could feasibly grow up to 2x the threshold in the slowstart code path. This is a bug that we fixed a while back in Windows TCP.